src/tasks/tasks.py

```diff
@@ -223,6 +223,11 @@ def n_train_examples(self):
     def n_val_examples(self):
         return self.example_counts['val']
 
+    @property
+    def no_train_data(self):
+        return self.get_split_text('train') == self.get_split_text('val') or \
```

Comment:
isn't this kind of expensive for what could otherwise be a Boolean we pass? I guess it does get kind of tricky if you want to have versions of the same task class that do and do not have training data, but I'm not sure how often that's happening

Reply:
I think that is not a big issue. Time-wise, this command is run only once, won't make any difference. Avoiding setting up another Boolean make it less likely for people to make mistakes when adding new tasks. As for the case of same task have and don't have training data, if the training split is set as convention (like mnli-diagnostic), this function will give us the correct Boolean.

Reply:
Hmm okay, fair enough.